README.md

```diff
@@ -20,12 +20,22 @@ Clone this repo into your ~/Sites folder, as "_hal"
 
 - LOCAL_HOWARD_D8_FOLDERS[0] = Your local hud8 root folder.
 - LOCAL_HOWARD_D8_FOLDERS[1] = Your local academicdepartments root folder.
+- E_MAIL = Your Acquia E-mail.
+- PRIVATE_KEY= Acquia Private Key.
```

Comment:
Maybe name these ACQUIA_EMAIL and ACQUIA_PRIVATE_KEY (would also need to adjust below) so its more clear. Also, need to add these to hal_config.default.txt so that people have that as a starter.

Reply:
You changed in the script, but this should also be ACQUIA_EMAIL, and ACQUIA_PRIVATE_KEY

Reply:
Correction made @drogers98